### DISCUSSION TITLE
When i will get my student develpoer pack ?

### DISCUSSION THREAD

**Discussion by Fodenjames** [community/community#50072] [Actions]
### Select Topic Area

Show & Tell

### Body

Hey 
I just want to know that how much days it will to confirm Me for student developer pack
Whenever I tried to give my student I'd it's shows it was created to recently so I want to know how much days it will take ?

**Comment by maahaMaya** [comment]
You can follow these link
[https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)

**Comment by Fodenjames** [comment]
So what can I do now?
If it's shows your account was created recently but it has been created 4
days ago

**Comment by Fodenjames** [comment]
What do you mean? I don't get it
